src/Razor/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/SingleLineControlFlowStatements_InCodeBlock/TestComponent.codegen.cs

```diff
@@ -45,7 +45,7 @@ protected override void BuildRenderTree(Microsoft.AspNetCore.Components.Renderin
 #line default
 #line hidden
 #nullable disable
-                                      
+                          
```

Comment:
@TanayParikh It looks like Ajay was seeing the same issue in https://github.com/dotnet/aspnetcore-tooling/pull/1606. It doesn't look too impactful, but just want to check this change is OK?

Reply:
Hmmm, so this wont have any actual impact on the runnability of the code since it's outside of the line pragma; however, it's concerning that this is being touched at all? Definitely need to look into what's happening here. Also, in the other files I wouldn't imagine that generated locations would change unless there was a mashup of tabs in the source files / interesting settings in the tests.

Reply:
Also, @pranavkm does this not worry you?